modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}-serial-console_{context}"]
+= Modifying a live install {boot-media} to enable the serial console
+
+By default, the serial console is disabled and all output is written to the graphical console. You can enable the serial console with the following procedure.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+ifeval::["{boot-media}" == "ISO image"]
+. Retrieve the {op-system} ISO image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and run the following command to customize the ISO image to enable the serial console to receive output:
++
+[source,terminal]
+----
+$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso \
+  --dest-console ttyS0 \ <1>
+  --dest-device /dev/sda <2>
+----
++
+<1> The desired primary console. In this case, the serial console.
```

Comment:
We should have more text in this section explaining the syntax of the argument.  In particular, a bare `ttyS0` defaults to 9600 bps (because that's the kernel default), which is probably not desired.